issue_test.go

```diff
@@ -712,6 +712,41 @@ func TestIssueService_GetCustomFields(t *testing.T) {
 	}
 }
 
+func TestIssueService_GetCustomFieldValues(t *testing.T) {
+	setup()
+	defer teardown()
+	testMux.HandleFunc("/rest/api/2/issue/createmeta", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		if r.URL.String() == "/rest/api/2/issue/createmeta?projectKeys=EX&issuetypeNames=Story&expand=projects.issuetypes.fields" {
+			w.WriteHeader(http.StatusOK)
+			fmt.Fprintf(w, `{"expand":"projects","projects":[{"name":"EX","issuetypes":[{"name":"Story","fields":{"customfield_10002":{"name":"EX-2","key":"customfield_10002","allowedValues":[{"value":"Value1","id":"1","children":[{"value":"Value11","id":"11"},{"value":"Value12","id":"12"},{"value":"Value13","id":"13"}]},{"value":"Value2","id":"2"},{"value":"Value3","id":"3"}]}}}]}]}`)
+		}
```

Comment:
Could you expand this into a multi-line string to improve readability of the object?